main/options.c

```diff
@@ -2417,7 +2417,7 @@ static parametricOption ParametricOptions [] = {
 	{ "_list-roles",            processListRolesOptions,        TRUE,   STAGE_ANY },
 	{ "maxdepth",               processMaxRecursionDepthOption, TRUE,   STAGE_ANY },
 	{ "options",                processOptionFile,              FALSE,  STAGE_ANY },
-	{ "output-format",          processOutputFormat,            TRUE,   STAGE_ANY},
+	{ "output-format",          processOutputFormat,            TRUE,   STAGE_ANY },
```

Comment:
Looking at https://github.com/universal-ctags/ctags/blob/master/main/options.c#L2420 there is no space after STAGE_ANY. I can of course remove the commit if you like